crates/valence_protocol/src/text.rs

```diff
@@ -829,6 +830,15 @@ impl<'a> From<&'a Text> for Cow<'a, Text> {
     }
 }
 
+impl From<Text> for Value {
+    fn from(value: Text) -> Self {
+        Value::String(
+            serde_json::to_string(&value)
+                .unwrap_or_else(|err| panic!("failed to jsonify text {value:?}\n{err}")),
```

Comment:
Hmmm I forgot that Text -> Json is a fallible operation... this isn't so great then. Maybe `Text` could be redesigned in a way to avoid this problem.

Reply:
It might not actually be possible for `Text` to panic when writing to a `String` buffer so it's fine I guess.

Reply:
It really shouldn't be able to happen I think